packages/features/bookings/lib/service/RegularBookingService.ts

```diff
@@ -2045,6 +2047,42 @@ async function handler(
       }
     : undefined;
 
+  const bookingFlowConfig = {
+    isDryRun,
+  };
```

Comment:
In one of my follow PRs I move, _bookingFlowConfig_ generation to validation service that validates all the data and returns a very readable format that has all the validated data.